### PR TITLE
Release 4.5.0

### PR DESCRIPTION
**Pull request by davimacedo**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,26 @@
...
+- NEW: Add LDAPS-support to LDAP-Authcontroller. [#7014](https://github.com/parse-community/parse-server/pull/7014). Thanks to [Fabian Strachanski](https://github.com/fastrde).
+- FIX: (beforeSave/afterSave): Return value instead of Parse.Op for nested fields. [#7005](https://github.com/parse-community/parse-server/pull/7005). Thanks to [Diamond Lewis](https://github.com/dplewis).
+- FIX: (beforeSave): Skip Sanitizing Database results. [#7003](https://github.com/parse-community/parse-server/pull/7003). Thanks to [Diamond Lewis](https://github.com/dplewis).
+- FIX: Fix includeAll for querying a Pointer and Pointer array. [#7002](https://github.com/parse-community/parse-server/pull/7002). Thanks to [Corey Baker](https://github.com/cbaker6).
+- FIX: Add encryptionKey to src/options/index.js. [#6999](https://github.com/parse-community/parse-server/pull/6999). Thanks to [dblythy](https://github.com/dblythy).
+- IMPROVE: Update PostgresStorageAdapter.js. [#6989](https://github.com/parse-community/parse-server/pull/6989). Thanks to [Vitaly Tomilov](https://github.com/vitaly-t)
```
Comment (TomWFox) [inline CHANGELOG.md:23]:
```suggestion
- IMPROVE: Update PostgresStorageAdapter.js. [#6989](https://github.com/parse-community/parse-server/pull/6989). Thanks to [Vitaly Tomilov](https://github.com/vitaly-t).
```

**Review by TomWFox — APPROVED** [review]
LGTM 👍

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,7 +1,26 @@
...
+- FIX: Definitions for accountLockout and passwordPolicy. [#7040](https://github.com/parse-community/parse-server/pull/7040). Thanks to [dblythy](https://github.com/dblythy).
+- FIX: Fix typo in server definitions for emailVerifyTokenReuseIfValid. [#7037](https://github.com/parse-community/parse-server/pull/7037). Thanks to [dblythy](https://github.com/dblythy).
+- SECURITY FIX: LDAP auth stores password in plain text. See [GHSA-4w46-w44m-3jq3](https://github.com/parse-community/parse-server/security/advisories/GHSA-4w46-w44m-3jq3) for more details about the vulnerability and [da905a3](https://github.com/parse-community/parse-server/commit/da905a357d062ab4fea727a21eac231acc2ed92a) for the fix. Thanks to [Fabian Strachanski](https://github.com/fastrde).
+- NEW: Reuse tokens if they haven't expired. [#7017](https://github.com/parse-community/parse-server/pull/7017). Thanks to [dblythy](https://github.com/dblythy).
+- FIX: Consistent casing for afterLiveQueryEvent. [#7023](https://github.com/parse-community/parse-server/pull/7023). Thanks to [dblythy](https://github.com/dblythy).
```
Comment (dplewis) [inline CHANGELOG.md:17]: Isn't this a breaking change?

Comment (davimacedo) [reply CHANGELOG.md:17]: Yes. It is. It was a feature introduced in 4.4.0 though. Should we go to 5.0.0 anyways?

Comment (dplewis) [reply CHANGELOG.md:17]: I think we can stick to 4.5.0. I foresee 5.0.0 being a massive release.

Comment (davimacedo) [reply CHANGELOG.md]: @dplewis I improved the note. What do you think?

Comment (dplewis) [reply CHANGELOG.md]: Can you move it to the top to make it more noticeable?

Comment (davimacedo) [reply CHANGELOG.md]: done

**Comment by dplewis** [comment]
Can you add Github Actions for the release and docs?

**Comment by davimacedo** [comment]
done

**Review by dplewis — APPROVED** [review]

**Pull request merged**